htp/htp_table_private.h

```diff
@@ -62,7 +62,7 @@ enum htp_table_alloc_t {
 
 struct htp_table_t {
     /** Table key and value pairs are stored in this list; name first, then value. */
-    htp_list_t *list;
+    htp_list_t list;
```

Comment:
I think this is going to break the ABI?

Reply:
I don't think so. This is in htp_table_private.h so this is not public (the public structure is an opaque pointer) and we still use the functions the same way such as `htp_table_create`

We do add some functions to the ABI `htp_list_init` and `htp_list_release` but that remains compatible with previous functions.

Reply:
Agreed. I tested it as well.